jsapp/js/components/activity/activity.constants.ts

```diff
@@ -239,7 +237,10 @@ export interface ActivityLogsItem {
     latest_deployed_version_id?: string;
     latest_version_id?: string;
     version_uid?: string;
-    second_user?: string;
+    username?: string;
     // a lot of more optional metadata props…
```

Comment:
Well...I thought about it when editing the file TBH... and in fact there's a whole bunch of other optional props there, it's just that we're not using them in FE for anything. 🤷🏻  

But yeah...let's remove it to avoid confusion.